package.json

```diff
@@ -39,6 +39,17 @@
       "editor/title/context": [{
         "command": "duplicate.execute"
       }]
+    },
+    "configuration": {
+      "type": "object",
+      "title": "VSCode-Duplicate configuration",
```

Comment:
Please, replace title field to `Duplicate Action Configuration` for consistency with other plugins.